Explorer/Assets/DCL/Passport/Configuration/PassportAdditionalFieldsConfiguration.asset

```diff
@@ -51,7 +51,7 @@ MonoBehaviour:
     logo: {fileID: 21300000, guid: 1b4d0df87e5e94b5bb432204bb1d26c6, type: 3}
     editionValues: {fileID: 11400000, guid: 5ac76ba89a93911468d5218dc64b3258, type: 2}
   - type: 9
-    title: HOBBIES
+    title: FAVORITE HOBBY
```

Comment:
is it a single hobby or multiple hobbies? could it be favorite hobbies?

Reply:
I dont know, i guess @RominaMarchetti modified this

Reply:
Karo made me change it. From now on will be a single hobby.